index.js

```diff
@@ -0,0 +1,16 @@
+import { createRoot } from 'react-dom/client';
+
+import { Header } from './src/header';
+import { Main } from './src/main';
+import { Footer } from './src/footer';
+
+const root = createRoot( document.getElementById('app') );
+const component = (
```

Comment:
- should be named with a Uppercase letter
- component is too abstract name (almost everything is component in you application)